src/Microsoft.ML.Core/Data/ISchemaBindableMapper.cs

```diff
@@ -63,13 +68,18 @@ public interface ISchemaBoundRowMapper : ISchemaBoundMapper, IRowToRowMapper
     /// so to rebind, the same input column names must be used.
     /// Implementing of this object are typically created using a definie input <see cref="ISchema"/>.
```

Comment:
>definie [](start = 66, length = 7)

[https://en.wiktionary.org/wiki/d%C3%A9finie](https://en.wiktionary.org/wiki/d%C3%A9finie)
``
French
Verb
définie

feminine singular of the past participle of définir``

defined probably